resources/js/components/content/IntroSlide.vue

```diff
@@ -0,0 +1,38 @@
+<template>
```

Comment:
Brian and I are moving all of our templates into `resources/js/components/modules/templates/IntroSlideTemplate.vue`